### ISSUE TITLE
Glossary terms and tags with accent marks (áéíóúÁÉÍÓÚ) in name cannot be assigned to entities in UI.

### ISSUE THREAD

**Issue by jromandv** [open-metadata/OpenMetadata#12418]
**Affected module**
UI, glossary and classifications

**Describe the bug**
Glossary terms and tags with accent marks (áéíóúÁÉÍÓÚ) cannot be assigned to entities in UI. If they we assigned before they appear in the UI but when trying to delete and assign a new one results in an error.

**To Reproduce**
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/8abf9dd5-9747-4e9d-acf9-7c296629b18f)
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/e1965cc3-1719-4023-bd20-ef2d2b387616)
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/c9d20619-2550-4141-8db1-8feff6f82f02)
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/65e6a769-8196-4f34-9255-e976b6fd438e)
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/3555a4d8-e68e-4af2-ac0b-48066bad7ab2)

**Expected behavior**

**Version:**
OM 1.1.0
Python 3.8

**Additional context**

**Comment by Sachin-chaurasiya** [comment]
Hello @jromandv, I have tried the same, and it works as expected.

https://github.com/open-metadata/OpenMetadata/assets/59080942/a694e12b-1111-45d4-81d6-71b3d8bc4b7f

**Comment by jromandv** [comment]
Hi @Sachin-chaurasiya, @sureshms 

I am now somewhat confused with this issue. 

After the upgrade to 1.1.0 I we recreated indexes again in elastic, for all the repository entities. 

- This morning after reading your comment I tried again to assign glossary terms at table and column level and it didn't work. 
- I have reindexed the entities, but without creating the objects again, and only for the glossary and tags. 
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/53c9d645-6c88-4598-bf69-78a88f1c9c11)

- Now the assignment of terms and tags with accent in the name at table and column level works .....
![image](https://github.com/open-metadata/OpenMetadata/assets/112470575/e85fbdd3-0c07-4166-9572-6d72dfb716c0)

After OM upgrade and when you recreate the indexes later for all the entities, I would not have to do additional reindexing... that's what I think.

Could it be that when reindexing all entities it's not really doing it for all?. 

It is the only reason that I find from my side .......
